Tests/Shared/SampleData/SampleDataFilesUtil.swift

```diff
@@ -0,0 +1,110 @@
+import Common
+import Foundation
+import XCTest
+
+// Class that interacts with static files stored in SampleDataFiles resources directory.
+//
+// Instead of storing long Swift Strings full of data, we store static files in a directory in the Tests/ directory.
+// Then, we can read in those files as Strings and use them in test functions.
+//
+// One common use case of using these static files is to save the file into the FileSystem of the app running during the
+// test.
+// That means that at runtime in tests, when `FileManager` in the SDK runs, it can read files just like it's running an
+// an iOS app.
+//
+// In order to obtain sample files:
+// 1. Plug iPhone into your Computer.
+// 2. Run a sample iPhone app to make it save files that you care about into the app's file storage.
+// 3. Open Xcode > Window > Devices and Simulators > Select your iPhone that's plugged in > Select app in Installed App
+// section > Select "..." icon > Download container.
+// 4. You can now browse the files saved to the app. Copy and paste files from there that you care about.
+public class SampleDataFilesUtil {
```

Comment:
I am not clear with the use of this class and it's functions. Where are the files getting saved being used ?